index.d.ts

```diff
@@ -0,0 +1,794 @@
+/*
+ * Copyright (c) 2015-2024 Snowflake Computing Inc. All rights reserved.
+ */
+
+/**
+ * The snowflake-sdk module provides an instance to connect to the Snowflake server
+ * @see [source](https://docs.snowflake.com/en/developer-guide/node-js/nodejs-driver)
+ */
+export const enum RowMode {
+    ARRAY = 'array',
+    OBJECT = 'object',
+    OBJECT_WITH_RENAMED_DUPLICATED_COLUMNS = 'object_with_renamed_duplicated_columns',
+}
+
+export const enum LogLevel {
+    ERROR = 'ERROR',
+    WARN = 'WARN',
+    INFO = 'INFO',
+    DEBUG = 'DEBUG',
+    TRACE = 'TRACE',
+}
+
+export const enum DataType {
+    String = 'String',
+    Boolean = 'Boolean',
+    Number = 'Number',
+    Date = 'Date',
+    JSON = 'JSON',
+    Buffer = 'Buffer',
+}
+
+declare enum StatementStatus {
```

Comment:
We do not need to export these enums because the user does not need to use them on the user side.  I only export the enums that the user needs. They do not need Error codes, StatementStatus, or QueryStatus when using the driver.

Reply:
Ok. It could stay like you propose